apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/codegen/FragmentType.kt

```diff
@@ -81,6 +67,14 @@ internal fun CodeGenerationAst.FragmentType.implementationTypeSpec(generateAsInt
       .typeSpec()
       .toBuilder()
       .addSuperinterface(GraphqlFragment::class)
+      .addSuperinterface(Adaptable::class.asTypeName().parameterizedBy(defaultImplementationType.typeRef.asTypeName()))
+      .addFunction(
+          FunSpec.builder("adapter")
+              .addModifiers(KModifier.OVERRIDE)
+              .returns(ResponseAdapter::class.asTypeName().parameterizedBy(defaultImplementationType.typeRef.asTypeName()))
```

Comment:
nit: 
`this.defaultImplementationType.typeRef.asAdapterTypeName()`

Reply:
This is actually not the same:

1.
```
override fun adapter(): ResponseAdapter<Data>
```

2.
```
override fun adapter(): TestQuery_ResponseAdapter
```

I think 1. is cleaner in the generated model, the `_ResponseAdapter` file being an implementation detail?